src/Core/MailTemplates/Handlebars/Welcome.text.hbs

```diff
@@ -1,35 +1,39 @@
 ﻿{{#>FullTextLayout}}
-Thank you for creating an account with Bitwarden. You may now log in with your new account.
+Welcome to Bitwarden and thank you for creating an account! Now you can extend robust security to all of your online experiences and devices. 
 
-Did you know that Bitwarden is free to sync with all of your devices? Download Bitwarden today on:
+Note: Your Master Password is the only way you can unlock the Vault and only you hold the key. Memorize it, or write it down and keep it safe.
 
-Desktop
+
+Get Started: Install Bitwarden
 ============
 
-Access Bitwarden on Windows, macOS, and Linux desktops with our native desktop application.
-https://bitwarden.com/#download
+You can access the Bitwarden Vault from anywhere and any device at the web vault ({{WebVaultUrl}}/?utm_source=welcome_email&utm_medium=email). You can also download and install Bitwarden on any desktop, device and browser (http://www.bitwarden.com/download).
+
 
-Web Browser
+Download Options
 ============
 
-Integrate Bitwarden directly into your favorite browser. Use our browser extensions for a seamless browsing experience.
-https://bitwarden.com/#download-browser
+http://www.bitwarden.com/download
 
-Mobile
+
+Securely Share with Bitwarden Organizations
 ============
 
-Take Bitwarden on the go with our mobile apps for your phone or tablet device.
-https://bitwarden.com/#download-mobile
+Bitwarden makes it easy to securely share passwords for teams and enterprises through Organizations. Join an Organization if invited, or launch a new one anytime from the Web Vault ({{WebVaultUrl}}/?utm_source=welcome_email&utm_medium=email) with the + New Organization button.
+
 
-Web
+Login to Bitwarden
 ============
 
-Stuck without any of your devices? Using a friend's computer? You can access your Bitwarden vault from any web enabled device by using the web vault.
 {{WebVaultUrl}}/?utm_source=welcome_email&utm_medium=email
 
 
-If you have any questions or problems you can email us from our website at: https://bitwarden.com/contact/?utm_source=welcome_email&utm_medium=email
+We're here for you
+============
+
+Check out our Help (http://www.bitwarden.com/help) site for documentation, join the Bitwarden Community forums (https://community.bitwarden.com/) and connect with other enthusiasts on Reddit (https://www.reddit.com/r/Bitwarden/). If you have any questions or issues, contact us (http://www.bitwarden.com/contact).
+
 
-Thank you!
+Happy safe and securing sharing,
```

Comment:
I don't understand this. Supposed to be `Happy safe and secure sharing`, maybe?

It still seems like a weird sign off for us unless the top thing to be pushed is organizations.

Reply:
Verbiage came from Marketing department. I can tag @elizabethbaier and @go12 for some feedback though!

Reply:
@MGibson1 @vincentsalucci I haven't been able to reach @elizabethbaier but I agree the signoff should probably not apply to sharing since the email goes to everyone. I recommend: Stay safe and secure! The Bitwarden Team

Reply:
Other options:
Happy password management,
Stay safe,
Welcome to easy password management,
Preferences?

Reply:
@vincentsalucci Let's switch it out with "Stay safe and secure,"

Reply:
@vincentsalucci Alrighty. The GTM team synced and we think we should update the second sentence in the "Note" box to say:  Memorize it, or write it down and keep it in a safe and secure place. 

Let me know if this pushes the copy into 3 lines. If so, I'm going to try to trim it to be back to 2 lines within layout.

Reply:
Updated.

Reply:
Didn't see the latest message - will update the PR again soon.

Reply:
@elizabethbaier As discussed - removed "Note:" and adjusted final part of the statement to be "...and keep it in a safe place"